Kubernetes/legos/k8s_remove_pod_from_deployment/k8s_remove_pod_from_deployment.py

```diff
@@ -0,0 +1,63 @@
+##
+# Copyright (c) 2023 unSkript, Inc
+# All rights reserved.
+##
+import pprint
+
+from kubernetes import client
+from pydantic import BaseModel, Field
+
+class InputSchema(BaseModel):
+    pod_name: str = Field(
+        title="Pod Name",
+        description="K8S Pod Name"
+    )
+    namespace: str = Field(
+        title="Namespace",
+        description="K8S Namespace where the POD exists"
+    )
+
+def k8s_remove_pod_from_deployment(output):
+    if not output:
+        return
+    
+    pprint.pprint(output)
+
+
+def k8s_remove_pod_from_deployment(handle, pod_name: str, namespace: str):
```

Comment:
how is this lego removing the pod from deployment, its just adding some labels?

Reply:
<img width="823" alt="Screenshot 2023-03-31 at 7 44 02 AM" src="https://user-images.githubusercontent.com/87547684/229152783-b7d95470-5cf7-4d28-8dfb-d1db3ce30170.png">

The deployment label returned is the same as pod label, hence i went with the implementation as it is here.

Reply:
https://stackoverflow.com/questions/71981366/how-to-get-kubernetes-deployments-labels-when-a-new-pod-is-created-updated-in-cl This link too indicates that pod label is the same as deployment label

Reply:
Please let me know what other way there is to figure out deployment label of a pod.